src/Plugin/Field/FieldFormatter/ErrandServicesFormatter.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_tpr\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\SortArray;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceEntityFormatter;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\helfi_tpr\Entity\ChannelTypeCollection;
+use Drupal\helfi_tpr\Entity\ErrandService;
+use Drupal\helfi_tpr\Entity\Service;
+
+/**
+ * Field formatter to render errand service maps.
+ *
+ * @FieldFormatter(
+ *   id = "tpr_errand_servie_formatter",
```

Comment:
fix typo in the id - let's use better tpr_service_err_channel_list
Label: TPR - Errand Service Channels List